spec/models/manageiq/providers/vmware/infra_manager/provision_spec.rb

```diff
@@ -1,4 +1,5 @@
 describe ManageIQ::Providers::Vmware::InfraManager::Provision do
+  include EvmSpecHelper::EmsFolder
```

Comment:
This should not be needed.   All of the things in spec/support are required up front.